src/components/Navbar.js

```diff
@@ -249,7 +249,7 @@ class Navbar extends React.Component {
     }
 
     onItemClick(bookName) {
-        
+        this.goToTab(2)
```

Comment:
How about we call this navigation function after setting the variables first? It's just usually a better idea.

Reply:
Yes, better to initialize before call the nav function. I will do that change.